uqcsbot/scripts/holiday.py

```diff
@@ -58,7 +59,7 @@ def get_holiday() -> Holiday:
     return choice(holidays_today) if holidays_today else None
 
 
-def get_holidays_from_page(holiday_page) -> list:
+def get_holidays_from_page(holiday_page) -> List[Holiday]:
```

Comment:
Sorry this was my bad for missing it, but what exactly is `holday_page`? Will likely need a type, too.